x-pack/solutions/observability/test/functional_solution_sidenav/tests/observability_sidenav.ts

```diff
@@ -57,6 +57,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         }
 
         // open Infrastructure popover and navigate to some link inside the panel
+        await solutionNavigation.sidenav.expandMore();
```

Comment:
We removed this in the original grid PR because there was more space in the sidenav so we didn't need to click "more" here. 
Now we need to revert this

https://github.com/elastic/kibana/pull/243181/files#r2546679194